### DISCUSSION TITLE
How to check if an account exists?

### DISCUSSION THREAD

**Discussion by suhanoves** [ecederstrand/exchangelib#1030] [Q&A] [answered]
When I create an account ```my_account = Account()```, the library does not send a request to the server. Lazy evaluation is good, but I have to catch the `ErrorNonExistentMailbox` in several parts of my code.

What is the cheapest way to immediately check whether such an account exists on the server or not?

**Comment by ecederstrand** [comment, marked as answer]
Cheapest way is probably to access the root folder, i.e. `account.root`
